--- ./unix/Makefile.gtk.org	Sun Aug 31 01:30:10 2003
+++ ./unix/Makefile.gtk	Sun Aug 31 13:49:13 2003
@@ -548,18 +548,19 @@
 	rm -f *.o plink pscp psftp pterm putty puttytel
 
 install:
-	$(INSTALL_PROGRAM) -m 755 plink $(bindir)/plink
-	$(INSTALL_PROGRAM) -m 755 pscp $(bindir)/pscp
-	$(INSTALL_PROGRAM) -m 755 psftp $(bindir)/psftp
-	$(INSTALL_PROGRAM) -m 755 pterm $(bindir)/pterm
-	$(INSTALL_PROGRAM) -m 755 putty $(bindir)/putty
-	$(INSTALL_PROGRAM) -m 755 puttytel $(bindir)/puttytel
-	$(INSTALL_DATA) -m 644 plink.1 $(man1dir)/plink.1
-	$(INSTALL_DATA) -m 644 pscp.1 $(man1dir)/pscp.1
-	$(INSTALL_DATA) -m 644 psftp.1 $(man1dir)/psftp.1
-	$(INSTALL_DATA) -m 644 pterm.1 $(man1dir)/pterm.1
-	$(INSTALL_DATA) -m 644 putty.1 $(man1dir)/putty.1
-	$(INSTALL_DATA) -m 644 puttytel.1 $(man1dir)/puttytel.1
+	$(INSTALL) -d $(DESTDIR)$(bindir) $(DESTDIR)$(man1dir)
+	$(INSTALL_PROGRAM) -m 755 plink $(DESTDIR)$(bindir)/plink
+	$(INSTALL_PROGRAM) -m 755 pscp  $(DESTDIR)$(bindir)/pscp
+	$(INSTALL_PROGRAM) -m 755 psftp $(DESTDIR)$(bindir)/psftp
+	$(INSTALL_PROGRAM) -m 755 pterm $(DESTDIR)$(bindir)/pterm
+	$(INSTALL_PROGRAM) -m 755 putty $(DESTDIR)$(bindir)/putty
+	$(INSTALL_PROGRAM) -m 755 puttytel $(DESTDIR)$(bindir)/puttytel
+	$(INSTALL_DATA) -m 644 plink.1 $(DESTDIR)$(man1dir)/plink.1
+	$(INSTALL_DATA) -m 644 pscp.1 $(DESTDIR)$(man1dir)/pscp.1
+	$(INSTALL_DATA) -m 644 psftp.1 $(DESTDIR)$(man1dir)/psftp.1
+	$(INSTALL_DATA) -m 644 pterm.1 $(DESTDIR)$(man1dir)/pterm.1
+	$(INSTALL_DATA) -m 644 putty.1 $(DESTDIR)$(man1dir)/putty.1
+	$(INSTALL_DATA) -m 644 puttytel.1 $(DESTDIR)$(man1dir)/puttytel.1
 
 install-strip:
 	$(MAKE) install INSTALL_PROGRAM="$(INSTALL_PROGRAM) -s"
