diff -burN putty-0.53b-2003-12-31.orig/unix/Makefile.gtk putty-0.53b-2003-12-31/unix/Makefile.gtk
--- putty-0.53b-2003-12-31.orig/unix/Makefile.gtk	2003-12-31 17:48:22.000000000 +0100
+++ putty-0.53b-2003-12-31/unix/Makefile.gtk	2003-12-31 17:50:10.000000000 +0100
@@ -549,6 +549,7 @@
 	rm -f *.o plink pscp psftp pterm putty puttytel
 
 install:
+	$(INSTALL) -d $(DESTDIR)$(bindir) $(DESTDIR)$(man1dir)
 	$(INSTALL_PROGRAM) -m 755 plink $(DESTDIR)$(bindir)/plink
 	$(INSTALL_PROGRAM) -m 755 pscp $(DESTDIR)$(bindir)/pscp
 	$(INSTALL_PROGRAM) -m 755 psftp $(DESTDIR)$(bindir)/psftp
